lib/languages/french.sty

```diff
@@ -0,0 +1,74 @@
+\ExplSyntaxOn
+% Use tilde (~) for spaces.
+
+% French captions
+\addto\captionsfrench
+  {
+    % Monster basics
+    \renewcommand\armorclassname{Classe~d'armure}
+    \renewcommand\hitpointsname{Hit~Points}
+    \renewcommand\speedname{Vitesse}
+    % Monster ability scores
+    \renewcommand\strstatname{FOR}
+    \renewcommand\dexstatname{DEX}
+    \renewcommand\constatname{CON}
+    \renewcommand\intstatname{INT}
+    \renewcommand\wisstatname{SAG}
+    \renewcommand\chastatname{CHA}
+    % Monster details
+    \renewcommand\skillsname{Compétences}
+    \renewcommand\dimmname{Immunité~contre~les~dégâts}
+    \renewcommand\dvulname{Vulnerabilité~aux~dégâts}
+    \renewcommand\dresname{Résistance~aux~dégâts}
+    \renewcommand\cimmname{Immunité~contre~les~états}
+    \renewcommand\savesname{Jets~de~sauvegarde}
+    \renewcommand\sensesname{Sens}
+    \renewcommand\defaultsensesname{Perception~passive}
+    \renewcommand\languagesname{Langues}
+    \renewcommand\challengename{Dangerosité}
```

Comment:
In the community translation of SRD5e (from aidedd.org, which I beleive is based on the official Black Book translation) this is called « Puissance »

Unfortunately I don't have the paperbacks of the official translations so this might be based on an old translation.

Reply:
In the french Monster Manual, the challenge field is labeled "Dangerosité".
![image](https://user-images.githubusercontent.com/674589/103204348-74e1a900-48f7-11eb-8763-5e82913f0804.png)

Reply:
I need to find me a translation!

Don't hesitate to resolve all the threads you have official sources for.